### PR TITLE
review comments for man page

### PR DESCRIPTION
**Pull request by holger-dengler**
This PR includes review comments. Beside typos and other minor corrections, it also contains a reference to non-standard conform usage of the initialization arguments.

### CODE REVIEW

File: man/pkcs11sign.cnf.5 [hunk 1]
```diff
@@ -82,6 +77,22 @@ default provider as forward.
...
+during initialization.
+.IP
+Note: The PKCS#11 standard v3.0 specifies that the initialization of a
+Cryptoki module fails if
+.IR pReserved
+is not a NULL_PTR. This parameter will only work with Cryptoki modules
+(e.g.  libnss) which does not implement this strict behavior. Cryptoki
+modules, which strictly implements the PKCS#11 standard v3.0 will fail
+if this parameter is set.
+.PP
```
Comment (hfreude) [inline man/pkcs11sign.cnf.5:95]: You mixed singular and plural here. I would recomment this wording:

This parameter will only work with Cryptoki modules (e.g.  libnss) which do not implement this strict behavior.
A Cryptoki module, which strictly implements the PKCS#11 standard will fail on C_Initialize with CKR_ARGUMENTS_BAD if this parameter is set.

Comment (holger-dengler) [reply man/pkcs11sign.cnf.5:95]: ok

**Comment by holger-dengler** [comment]
Update:
- include review comments and add another typo-fix I forgot to add to the PR.

**Comment by holger-dengler** [comment]
Update:
- include another set of typos.

**Review by hfreude — APPROVED** [review]

**Pull request merged**